### DISCUSSION TITLE
How to set up autocompletion in Atom

### DISCUSSION THREAD

**Discussion by ThePhilgrim** [atom/atom#22441] [Q&A]
Hi, 
I am getting quite annoying autocompletion, even when writing comments and just regular strings. For example, when writing a comment "# .... BNC here", "here" gets automatically changed to "Otherwise" when hitting enter.

![Screenshot 2021-05-16 at 14 28 36](https://user-images.githubusercontent.com/76887896/118397266-70721e80-b653-11eb-8a09-f98207ec6324.png)

Is it possible to set up autocomplete-plus in a better way to only work with variables, and (programming) language-specific syntax?

If not, is it possible to disable Atom's autocompletion but keeping the Kite autocompletion enabled? I notice that if I disable autocomplete-plus, the Kite autocompletion gets disabled too.

Thanks for help!

**Comment by icecream17** [comment]
This also happens in vscode - my terrible workaround was to use "Tab" instead of "Enter" to autocomplete. Then you can just ignore it when autocomplete doesn't make sense

There's probably a better answer, just dropping this out

**Reply by ThatXliner** [reply]
Probably kite's fault. They added this as a feature, I think, (probably to compete with TabNine)
